docker/config-cluster/tasks/k8s-deployment-unconfig.yml

```diff
@@ -1,20 +1,20 @@
 ---
 
-- name: "check tiller installed"
-  shell: helm version --server --short
-  register: helm_installed
-  ignore_errors: true
-  changed_when: False
-
-- name: Remove all helm deployments
-  shell: helm list --short | xargs --max-lines=1 --no-run-if-empty helm delete
-  register: helm_output
-  when: helm_installed is succeeded
-
-- name: feedback
-  debug:
-    msg: "{{ helm_output.stdout_lines }}"
-  when: helm_installed is succeeded
+# - name: "check tiller installed"
```

Comment:
Questo potrebbe rimanere, così funziona in tutti i casi.  Se Helm 2 non c'è `helm_installed` diventa False e gli altri passi non vengono eseguiti

Reply:
Ok, darà un errore ma viene skippato e lo script continua.